KeenClientTests/KIODBStoreTests.m

```diff
@@ -45,6 +45,8 @@ - (void)testInit {
     NSString *dbPath = [self databaseFile];
     NSFileManager *fileManager = [NSFileManager defaultManager];
     XCTAssertTrue([fileManager fileExistsAtPath:dbPath], @"Database file exists.");
+    [store drainQueue];
```

Comment:
Can the `drainQueue`/`closeDB` operations be performed in `tearDown` or something similar?